BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -774,7 +760,6 @@ - (void)applicationWillEnterForeground:(UIApplication *)application {
 }
 
 - (void)appDidBecomeActive:(UIApplication *)application {
-    [self trackAppOpen];
```

Comment:
let us confirm with @aswani521 about the removal of this `app_open`

Reply:
Sure. I can update this later once we get confirmation from Aswani.